### ISSUE TITLE
Rename files in /docs beginning with "rmim"

### ISSUE THREAD

**Issue by morosebose** [v9y/giit#65]
The /docs directory has 21440 sequentially numbered songs, and 882 songs numbered "rmim0001" through "rmim0882". Renumber the latter from 21441 to 22322.

**Comment by v9y** [comment]
This is trickier than it seems. 
1. We will need to come up with a range that has no conflicts. 
2. I will need to update the giitaayan index database accordingly. Having an easy, programmable mapping between the current numbers and the newer number will make this task easier.

**Comment by morosebose** [comment]
I thought that the sequencing 21441–22322 would not have any conflicts, since the existing numbered songs are 1–21440? And I assume the songs with "rmim" are also sequential from 0001 through 0882?

**Comment by v9y** [comment]
Conflict-wise you are right. I was thinking though that we could use a pattern based on existing numbers so that the mapping (and changing the values in the index) is also easier. So something like, replacing the string 'rmim' with '3'. So, e.g., rmim0882 will be 30882 and rmim0001 will be 30001. This is a simple rule that can be applied on both sides - filename change and index field updates. What do you think?

**Comment by morosebose** [comment]
Sounds good to me. Would you like me to make the change on the filename side? I can issue the PR some time this week and you can merge whenever you're ready on the index field side.

**Comment by v9y** [comment]
We might have to coordinate this to do around the same time, otherwise it would break the songs/links on giitaayan. I am fine with the non-availability for a day or two though. Let me check what's exactly involved on the giitaayan side and get back to you.

**Comment by morosebose** [comment]
Well, they would break only if you merged the pull request. 😀 But yeah, best to coordinate. No rush.

**Comment by v9y** [comment]
Yes of course. Go ahead and do that. Better still, if it's possible, can you copy the files with new names  rather than renaming them (keeping the old ones intact). That will allow me to try things out. We will later remove the old ones.
